R-packages/covidcast/R/plot.R

```diff
@@ -104,6 +107,80 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
   geo = df$geo
   names(val) = geo
 
+  # Make background layer for MSA and HRR maps which are incomplete
+  if ((attributes(x)$geo_type == "msa") |
+      (attributes(x)$geo_type == "hrr")) {
+    map_df = sf::st_read('../data/shapefiles/state/cb_2019_us_state_5m.shp')
```

Comment:
Hmm. I think this (and the similar call below) relies on the R working directory being exactly right. This may not work when the user installs the package themselves. Should we be putting the shapefiles in `inst/` and using `system.file(path, package="covidcast")` or something like that?